src/cook-web/src/components/dnd-kit-sortable-tree/ui/folder/FolderTreeItemWrapper.tsx

```
@@ -1,112 +1,112 @@
+import './FolderTreeItemWrapper.css';
+
+function flattenParents<T>(
+  parent: FlattenedItem<T> | null,
+): FlattenedItem<T>[] {
+  if (!parent) return [];
+  return [...flattenParents(parent.parent), parent];
+}
+
+export const FolderTreeItemWrapper = forwardRef<
+  HTMLDivElement,
+  React.PropsWithChildren<TreeItemComponentProps<{}>>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)